libs/ledgerjs/packages/hw-bolos/src/loadPKI.ts

```diff
@@ -0,0 +1,54 @@
+import Transport from "@ledgerhq/hw-transport";
+import { handledErrorCode, throwError } from "./PKIError";
+
+const CLA = 0xb0;
+const INS = 0x06;
+const P2 = 0x00;
+
+// Extract from https://github.com/LedgerHQ/ledger-secure-sdk/blob/d59b21f8a1f2e78ad50b5a8a21d0daac68826643/include/os_pki.h#L74
+const keyUsage = {
+  GENUINE_CHECK: 0x01,
+  EXCHANGE_PAYLOAD: 0x02,
+  NFT_METADATA: 0x03,
+  TRUSTED_NAME: 0x04,
+  BACKUP_PROVIDER: 0x05,
+  RECOVER_ORCHESTRATOR: 0x06,
+  PLUGIN_METADATA: 0x07,
+  COIN_META: 0x08,
+  SEED_ID_AUTH: 0x09,
+  UNKNOWN: 0x0a,
+};
+
+const OkStatus = 0x9000;
```

Comment:
Should we use the code from the transport package ?

Reply:
I don't see it in the Transport package :/